docroot/themes/humsci/humsci_basic/patterns/gradient-hero/gradient-hero.html.twig

```diff
@@ -0,0 +1,23 @@
+{%- set attributes = attributes.addClass('hb-gradient-hero') -%}
+
+{% if variant %}
+  {% set variant_class = "hb-gradient-hero--content-#{variant}" %} 
+  {% set attributes = attributes.addClass(variant_class) %}
+{% endif %}
+
+{% spaceless %}
+  <div{{ attributes }}>
+    <div class="hb-gradient-hero__wrapper">
```

Comment:
As I was doing the styles, I found that this `div` with the `hb-gradient-hero__wrapper` class unnecessary. So it is removed in `STN-675--gradient-hero-styles` branch.